src/query/user.ts

```diff
@@ -1,12 +1,18 @@
 import { resolveUserUsingJWT } from "../utils/resolveUser";
-import { FragmentableArray, User } from "../generated/prisma-client";
+import { Admin, User } from "../generated/prisma-client";
 
-export const user = (parent, { username }, ctx): FragmentableArray<User> => {
+export const user = (parent, { username }, ctx): Promise<User> => {
   return ctx.prisma.user({
     username,
   });
 };
 
+export const adminUser = (parent, { username }, ctx): Promise<Admin> => {
+  return ctx.prisma.admin({
+    username,
+  });
+};
+
```

Comment:
I think this would return the encrypted password. Its fine, but not recommended right?

Reply:
yes, we don't want to add the password field, even if it is encrypted

Reply:
Do you want me to help resolve this with a new `type Credential` linked to `type Admin`?

Reply:
Let's take this to today's meeting too

Reply:
I'll work on this